### PR TITLE
Conversion from tileKey into extendedSpatialID

### PR DESCRIPTION
**Pull request by mi-24v**
## feature

- tileKey->extendedSpatialId conversion via `ConvertQuadkeysAndAltitudekeysToExtendedSpatialIDs`
- documentation for `ConvertQuadkeysAndAltitudekeysToExtendedSpatialIDs`

### CODE REVIEW

File: transform/convert_quadkey_and_Vertical_id_test.go
Comment (HarutakaMatsumoto) [inline transform/convert_quadkey_and_Vertical_id_test.go:1]: Why do you change all of this?

**Review by HarutakaMatsumoto — CHANGES_REQUESTED** [review]
I reviewed it.
Check it out.

File: transform/convert_quadkey_and_Vertical_id_test.go
Comment (mi-24v) [reply transform/convert_quadkey_and_Vertical_id_test.go:1]: It seems to be CRLF -> LF conversion was made in [e5825d39](https://github.com/trajectoryjp/spatial_id_go/commit/e5825d39f233b801210b63e999f4a3d1b1dfcc53?diff=split&w=1)
using `ignore whitespace` will  suppress too many diff
need fix?

**Comment by mi-24v** [comment]
it founds that QuadkeysAndAltitudekeys should not be used in new feature (#28 )
This PR will be closed and replacement Pull Request is to considered.

**Pull request closed without merging**